doc/source/data/inspecting-data.rst

```diff
@@ -143,3 +144,53 @@ of the returned batch, set ``batch_format``.
 For more information on working with batches, see
 :ref:`Transforming batches <transforming_batches>` and
 :ref:`Iterating over batches <iterating-over-batches>`.
+
+
+Inspecting runtime statistics
+=============================
+
+Ray Data calculates statistics during execution like runtime and memory usage for the different stages. 
+
+View debug stats for your Dataset executions via :meth:`Dataset.stats() <ray.data.Dataset.stats>` on an executed dataset. The stats are also persisted under `/tmp/ray/session_*/logs/ray-data.log`.
+
+.. testcode::
+    import ray
+    import time
+
+    def pause(x):
+        time.sleep(.0001)
+        return x
+
+    ds = ray.data.read_csv("s3://anonymous@air-example-data/iris.csv")
+    ds = ds.map(lambda x: x)
+    ds = ds.map(pause)
```

Comment:
For consistency with the other code snippets in the user guides, let's chain the method calls?

(Not sure if the syntax in the suggestion is totally correct)
```suggestion
    ds = (
        ray.data.read_csv("s3://anonymous@air-example-data/iris.csv")
        .map(lambda x: x)
        .map(pause)
    )
```